src/stringify/stringifyString.js

```diff
@@ -204,7 +204,7 @@ function blockString({ comment, type, value }, ctx, onComment, onChompKeep) {
   return `${header}\n${indent}${body}`
 }
 
-function plainString(item, ctx, onComment, onChompKeep) {
+function plainString(item, ctx, onComment, onChompKeep, defaultQuote) {
```

Comment:
Why is `defaultQuote` being passed as an arg, rather than being read from `strOptions`?

Reply:
I did not notice that `strOptions` is available in this context. Will try to remove the parameter